### DISCUSSION TITLE
Apache auth framework

### DISCUSSION THREAD

**Discussion by aanderse** [latchset/mod_auth_mellon#68] [Ideas]
In https://github.com/UNINETT/mod_auth_mellon/issues/120#issuecomment-304843601 it was mentioned that `mod_auth_mellon` doesn't integrate with the `apache` auth framework (`Require ip`, `Require group`, etc...) and uses its own system for authz (`MellonCond`).

It would be really nice if `mod_auth_mellon` would integrate with the `apache` auth framework and support standard `apache` syntax, like other `apache` modules do. I understand that is like a large job, but is there any interest in this? Would pull requests making this change be accepted? Is there anyone on the development team who is interested in working on this? If there is enough interest would developer sponsorship be an option?

**Comment by thijskh** [comment]
I would personally definitely be in favour of a PR accomplishing this. Ideally it would remain backwards compatible. I have no idea about the amount of work required.
